docs/src/docs/modules/layout-check.adoc

```diff
@@ -12,15 +12,17 @@ Although, manual inspection can never be pixel-accurate. An inspection of perhap
 
 At this point an automatic comparison of reference screenshots can help. This test is to be seen in addition to the functional tests, but cannot replace them. In a test case, it would also be conceivable to combine the check of function and layout.
 
-== Description
+== How it works
 
 A layout test with the Testerra utilities is actually a comparison between a reference and the actual state. This is done via the screenshot functionality of Selenium, in which both screenshots (reference and actual) are compared pixel by pixel from top to bottom. In this comparison, a third image is created in which differences (ReferencePixel != ActualPixel) are marked in red.
 
```

Comment:
Is there a chance to add graphic with a reference image, an actual image and a difference image?